python/plugins/db_manager/db_plugins/gpkg/connector.py

```diff
@@ -620,6 +620,14 @@ def runVacuum(self):
         """ run vacuum on the db """
         self._execute_and_commit("VACUUM")
 
+    def commentTable(self, schema, tablename, comment=None):
+        """Comment the table"""
+        return ''
+
+    def getComment(self, tablename, field):
```

Comment:
Move this method `getComment` into the base class `DBConnector` as a default implementation and override in PG connector only, instead of providing the same default implementation in all connectors but the PG one.